common/lib/api-client/models.js

```diff
@@ -237,6 +237,11 @@ module.exports = {
       defaultInclude: ['suppliers'],
     },
   },
+  locations_free_spaces: {
+    options: {
```

Comment:
Are there any other model configuration options I should be including here? Default property names, types? It's a shallow data object as we only really care about the location name and the associated meta object with availability.

Reply:
`defaultInclude` is the main one that you might want to define. This is passed to the api client as the list of resources to include when making a request.

I'm guessing there might be some relationships with your call so you might want to define that. Alternatively, this can be defined in the services and then you don't need to worry about.

This work was done to allow the transition from v1 to v2 of the API where v1 always included everything by default and v2 included nothing. So this addition was to allow us to switch with less refactoring elsewhere.

Reply:
`locations_free_spaces.getLocationFreeSpaces` is very shallow as it's a jumping point onto other parts of the system:

_sample request url:_
`${host}/api/v1/locations_free_spaces?filter[location_type]=prison&filter[location_id]=${locationIds}&page=1&per_page=100&date_from=2020-11-02&date_to=2020-11-08`

so I don't think there's any relationships to be defined, at least at the moment.